Formula/x/xroar.rb

```diff
@@ -0,0 +1,37 @@
+class Xroar < Formula
+  desc "Dragon and Tandy 8-bit computer emulator"
+  homepage "https://www.6809.org.uk/xroar/"
+  url "https://www.6809.org.uk/xroar/dl/xroar-1.5.5.tar.gz"
+  sha256 "a39b319aa5d46f455e8973cf7f3b6da67f24231dc1c91fdcb3c09e7a689d3c8b"
+  license "GPL-3.0-or-later"
+
+  head do
+    url "https://www.6809.org.uk/git/xroar.git", branch: "master"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+    depends_on "texinfo" => :build
+  end
+
+  depends_on "libpng"
+  depends_on "libsndfile"
+  depends_on "sdl2"
+
+  uses_from_macos "zlib"
+
+  on_linux do
+    depends_on "alsa-lib"
+  end
+
+  def install
+    system "./autogen.sh" if build.head?
+    system "./configure", "--disable-silent-rules", "--without-x", *std_configure_args
+    system "make", "install"
+  end
+
+  test do
+    assert_equal "XRoar #{version}",
+                 shell_output("#{bin}/xroar -V").split("\n").first
```

Comment:
```suggestion
    assert_match version.to_s, shell_output(bin/"xroar -V")
```

Reply:
This is fine to include, but you'll need to add an additional test that actually verifies some functionality of the app.

Reply:
That will be a bit hard as the app is an emulator of a retro computer. Testing actual functionality would involve downloading ROM images that might be copyrighted material in some countries. I just followed the example from atari800 formula.

Reply:
What about matching something from `xroar -config-print`? That won't involve ROM images and will ensure at least the binary is outputting expected information beyond version.

Reply:
> What about matching something from `xroar -config-print`? That won't involve ROM images and will ensure at least the binary is outputting expected information beyond version.

I can certainly give it a go, but if e.g. new machines are added, that output might change in unexpected ways.

Reply:
No worries. If the output changes in a way that breaks the test, CI will catch it when we try to bump the version and we can fix it then.

Reply:
@p-linnane, just pushed a new version matching the machines in the output of `-config-print`. Do you think it looks better?